pkg/render/tiers/tiers_test.go

```diff
@@ -30,10 +30,40 @@ var _ = Describe("Tiers rendering tests", func() {
 
 	clusterDNSPolicy := testutils.GetExpectedPolicyFromFile("../testutils/expected_policies/dns.json")
 	clusterDNSPolicyForOCP := testutils.GetExpectedPolicyFromFile("../testutils/expected_policies/dns_ocp.json")
+	nodeLocalDNSPolicy_ipv4 := testutils.GetExpectedGlobalPolicyFromFile("../testutils/expected_policies/node_local_dns_ipv4.json")
```

Comment:
nit, but `_` isn't commonly used in variable names in Golang - these should be `nodeLocalDNSPolicyIPv4`, etc.